content/reference/rest-api/templates/update.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Templates: update'
+weight: 10
+
+menu:
+  main:
+    name: 'update'
+    identifier: 'rest-api-templates-update'
+    parent: 'rest-api-templates'
+    pre: 'Updates a template.'
+---
+
+Updates a template.
+
+# Request
+
+## HTTP Request
+
+```
+PATCH https://cawemo.com/api/v1/templates/:id
```

Comment:
We could also offer the publishing as an optional parameter for the create and update actions. This way, users would only need to make one call to update and publish a template.

Reply:
See https://github.com/camunda/camunda-docs-cawemo/pull/118/files#r687661461